docs/statistics/probability/General.md

```diff
@@ -61,9 +61,69 @@ Why is probability so important? Probability plays a central role in fields like
 </figure>
 
-We rely on probability whenever uncertainty exists about the outcome of an event. For example, determining whether men are generally taller than women requires probability because it isn't universally true that all men are taller than all women. Similarly, probability helps assess the accuracy of medical tests, as even highly reliable tests can produce false positives. In contrast, for questions with known answers, such as the speed of light versus the speed of sound, probability isn't needed.
+We rely on probability whenever uncertainty (e.g. randomness) exists about the outcome of an event. For example, determining whether men are generally taller than women requires probability because it isn't universally true that all men are taller than all women. Similarly, probability helps assess the accuracy of medical tests, as even highly reliable tests can produce false positives. In contrast, for questions with known answers, such as the speed of light versus the speed of sound, probability isn't needed.
 
-### Probability vs. Proportion
+## Random Variables
+
+A **random variable** differs slightly from traditional mathematical variables. In mathematics, variables can:
+
+1. Represent values that may vary.
+2. Act as unknowns to be solved.
+
+For random variables, the first property holds, but the second does not. Random variables allow us to perform **calculations** with the outcomes of a **random experiment**, making it possible to model and work with uncertainty. They are typically represented by capital letters, and their specific values are denoted by lowercase letters.
+
+???+ defi "Definition: Random Variable"
+    A random variable is a function that assigns numbers to the outcomes of a random process (mapping).
+
+
+For example, consider the result of rolling a die:
+
+\[
+X = \text{The number shown when a fair die is rolled}
+\]
+
+Or the result of flipping a coin:
+
+\[
+X = 
+\begin{cases}
+1, & \text{if heads} \\
+0, & \text{if tails}
+\end{cases}
+\]
+
+In both cases, the random variable maps the outcome of a random process to a numerical value. In the context of a coin flip:
+
+- The **random process** is the coin flip itself.
+- Each **experiment** refers to an individual flip.
+- An **event** is the outcome, such as getting heads or tails.
+- The **sample space** is the set of all possible outcomes (heads or tails).
+- The **random variable** assigns a numerical value (e.g., 1 for heads, 0 for tails) to each event.
+
+Thus, random variables enable us to quantify the outcomes of random processes, allowing for further analysis and interpretation.
+
+???+ question "Task"
+    Now it's your turn to create a random variable. There are several packages in Python that we can use for this purpose. Use the `random` package we already used in the package [management section](../../python/packages.md#standard-library). 
+
+    <figure markdown="span">
+    ![Correlation Types](/assets/statistics/meme_dice.jpg){width=50% }
```

Comment:
```suggestion
    ![Correlation Types](../../assets/statistics/meme_dice.jpg){width=50% }
```

When we're using Markdown we can use relative paths to get rid of these annoying warnings when serving the page:

```
INFO    -  Doc file 'statistics/probability/General.md' contains an absolute link
           '/assets/statistics/meme_dice.jpg', it was left as is. Did you mean
           '../../assets/statistics/meme_dice.jpg'?
```

If it's ok, I'll fix the links in a small commit? 😄

Reply:
that would be nice. thanks